Error: Peer certificate cannot be authenticated with given CA certificates.

Lastspass CLI has been working fine then suddenly today I get the following message when attempting login:

Error: Peer certificate cannot be authenticated with given CA certificates.

Strange, that didn't work for me.  It doesn't build with the new CA.

certificate.h:1:28: warning: missing terminating " character
\n"----END CERTIFICATE-----7z9M/P2C2F+fpErgUfCJzDupxBdN49cOSvkBPB7
                            ^
certificate.h:2:1: error: stray ‘\’ in program
"
"

--- a/pkgs/tools/security/lastpass-cli/default.nix
+++ b/pkgs/tools/security/lastpass-cli/default.nix
@@ -3,22 +3,18 @@
 stdenv.mkDerivation rec {
   name = "lastpass-cli-${version}";
 
-  version = "0.4.0";
+  version = "0.5.1";
 
   src = fetchFromGitHub {
     owner = "lastpass";
     repo = "lastpass-cli";
     rev = "v${version}";
-    sha256 = "1r06lifjc28sm88d6x3xwn76l9fjwjmd1wbmvr9j8rna889q0wl9";
+    sha256 = "0k2dbfizd6gwd4s8badm50qg2djrh22szd932l1a96mn79q8zb70";
   };
 
   buildInputs = [
     openssl curl libxml2 pkgconfig asciidoc docbook_xsl libxslt
   ];
-
-  prePatch = ''
-    substituteInPlace version.h --replace "0.3.0" "0.4.0"
-  '';
 
   makeFlags = "PREFIX=$(out)";
 

